src/api/app/models/bs_request.rb

```diff
@@ -52,6 +52,8 @@ class SaveError < APIException
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
 
+  belongs_to :user, foreign_key: 'creator', primary_key: 'login'
```

Comment:
No, you need to specify both, foreign_key and primary_key because otherwise rails will use ``id`` as primary_key and ``user_id`` as foreign key (Convention over Configuration...). But in BsRequest ``creator`` column we store the ``login`` and not the user id, so we need to overwrite the convention.